lib/request.js

```diff
@@ -191,11 +203,13 @@ Object.defineProperties(Request.prototype, {
   },
   routeConfig: {
     get () {
+      warning.emit('FSTDEP015')
       return this[kRouteContext][kPublicRouteContext].config
     }
   },
   routeSchema: {
     get () {
+      warning.emit('FSTDEP014')
```

Comment:
We need to test is as https://github.com/fastify/fastify/blob/main/test/listen.deprecated.test.js

Reply:
created a file ptal